notebooks/WORKSPACE

```diff
@@ -0,0 +1,102 @@
+workspace(name = "TRTorch")
```

Comment:
Can we rename outside the container to WORKSPACE.tpl or something, just in case it might confuse bazel.

Reply:
sure, let's do WORKSPACE.notebook. It's just a version that uses existing libraries in the container. If there's a parameter for Bazel to switch between  downloaded and pre-installed library I think that'd be cleaner yet.

Reply:
Yeah that's true we have been looking but havent found one yet. I havent found a great way to list multiple possible sources for dependencies